Classes/Models/Basic Models/Generated/CPProtoSpecification.h

```diff
@@ -109,8 +165,8 @@ extern NSString * const kCPSpecificationDefaultSubspecAttributeKey;
 /**
   The requires_arc attribute.
 */
-- (BOOL)requiresArc;
-- (void)setRequiresArc:(BOOL)requiresArc;
+- (NSString*)requiresArc;
```

Comment:
`requiresArc` probably used to be just a BOOL, but is a bit more expressive these days.

From [cocoapods-core/specification/dsl.rb](https://github.com/CocoaPods/Core/blob/master/lib/cocoapods-core/specification/dsl.rb#L653-L683)

```
      # @!method requires_arc=(flag)
      #
      #   `requires_arc` allows you to specify which source_files use ARC.
      #   This can either be the files which support ARC, or true to indicate
      #   all of the source_files use ARC.
      #
      #   Files which do not use ARC will have the `-fno-objc-arc` compiler
      #   flag.
      #
      #   The default value of this attribute is `true`.
      #
      #   @example
      #
      #     spec.requires_arc = false
      #
      #   @example
      #
      #     spec.requires_arc = 'Classes/Arc'
      #
      #   @example
      #
      #     spec.requires_arc = ['Classes/*ARC.m', 'Classes/ARC.mm']
      #
      #   @param [Bool, String, Array<String>] flag
      #          whether the source files require ARC.
      #
      attribute :requires_arc,
                :types => [TrueClass, FalseClass, String, Array],
                :file_patterns => true,
                :default_value => true,
                :inherited => true
```